money-akka/src/main/scala/com/comcast/money/akka/stream/TracedFlow.scala

```diff
@@ -39,9 +42,11 @@ abstract class TracedLogic[In, Out](shape: Shape)
     tracer.startSpan(key)
     val outMessage = stageLogic(inMessage)
     push[TracedOut](out, (outMessage, spanContext))
+    tracer.stopSpan()
```

Comment:
why is this stopping twice?

Reply:
Yeah so I made a mistake first time through which was to only close all the spans at the very end. Which I had thought from the previous PR was what was wanted but after discussing with Kris this made no sense as it was not actually tracking the time taken in this section of the flow. I will have to change the `traceStageAndStopAll` to make more sense.

Reply:
I just noticed the duplication yeah I fixed that just now